### PR TITLE
Epic design variations v3

### PR DESCRIPTION
**Pull request by Mullefa**
@guardian/contributions 

## What does this change?

Releases epic design variations version 3.

## What is the value of this and can you measure success?

To find a new version of the Epic with a higher conversion rate than the control.

## Screenshots

- __Control__

control-desktop

- __Bigger buttons__

bigger-buttons-desktop

- __Colour change__

colour-change-desktop

- __Paypal and credit card__

paypal-and-credit-card-desktop

## Tested in CODE?

No, tested locally.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js [hunk 1]
```diff
@@ -23,9 +23,10 @@ define([
         variants: ['control', 'fairness_strong', 'fairness_strong_alternate_hook']
 	};
 
-    var AcquisitionsEpicDesignVariationsV2 = {
-        name: 'AcquisitionsEpicDesignVariationsV2',
-        variants: ['control', 'highlight_subtle', 'highlight_perspective', 'highlight_secure', 'highlight_hard', 'paypal']
+    var AcquisitionsEpicDesignVariationsV3 = {
+        name: 'AcquisitionsEpicDesignVariationsV3',
+        // TODO
```
Comment (jranks123) [inline static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js]: Remove TODO

File: static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss [hunk 2]
```diff
@@ -1,57 +1,41 @@
...
-.contributions__title--epic--large {
-    @include font-size(36px);
-    @include mq(tablet) {
-        @include font-size(38px);
-    }
+.contributions__paragraph--highlight {
+    background-color: #ffe399;
+    padding: 4px 2px;
+    margin-left: -2px;
+    margin-right: -2px;
```
Comment (jranks123) [inline static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss]: $gs-baseline/-10

Comment (jranks123) [inline static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss]: (lol jkz)

**Review by jranks123 — APPROVED** [review]

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-design-variations-v3.js [hunk 3]
```diff
@@ -71,49 +75,47 @@ define([
...
     return contributionUtilities.makeABTest({
-        id: 'AcquisitionsEpicDesignVariationsV2',
-        campaignId: 'kr1_epic_design_variations_v2',
+        id: 'AcquisitionsEpicDesignVariationsV3',
+        campaignId: 'kr1_epic_design_variations_v3',
 
-        start: '2017-03-07',
-        expiry: '2017-04-20',
+        start: '2017-03-22',
+        expiry: '2017-04-30',
```
Comment (jranks123) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-design-variations-v3.js]: this is a Sunday

File: common/app/conf/switches/ABTestSwitches.scala [hunk 4]
```diff
@@ -202,9 +202,9 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-acquisitions-epic-design-variations-v2",
+    "ab-acquisitions-epic-design-variations-v3",
     "Test more design variations to the Epic",
-    owners = Seq(Owner.withGithub("desbo")),
+    owners = Seq(Owner.withGithub("Mullefa")),
     safeState = Off,
     sellByDate = new LocalDate(2017, 4, 20),
```
Comment (jranks123) [inline common/app/conf/switches/ABTestSwitches.scala]: this doesn't match with the date i the test

**Review by desbo — APPROVED** [review]

**Pull request merged**